src/core/core.scale.js

```diff
@@ -224,6 +224,82 @@ function skip(ticks, newTicks, spacing, majorStart, majorEnd) {
 
 class Scale extends Element {
 
+	constructor(cfg) {
+		super();
+
+		/** @type {string} */
```

Comment:
I don't suppose all of this gets stripped away in the prod build?

Reply:
The comments would. Not the variables